.github/workflows/test.yml

```diff
@@ -29,7 +29,7 @@ jobs:
   
       - name: Build code
         run: |
-          yarn bootstrap
```

Comment:
Why not just run `yarn bootstrap` in the last step?

Reply:
Yes... I guess it is customary to have an install step separate from build step. It makes the logs for each step easy to expand/close in the log viewer. But we can probably just have an `Install and Build Code Step`